pages/blog/swc-1.mdx

```diff
@@ -1,23 +1,27 @@
 ---
-title: Introducing swc 1.0
-author: DongYoon Kang
-authorURL: "http://github.com/kdy1"
-authorFBID: 100024888122318
+image: https://user-images.githubusercontent.com/9113740/140682117-5e9f460e-c8f1-447c-88a0-e75f80779260.png
+description: 'SWC (Speedy Web Compiler) is a super-fast JavaScript and TypeScript compiler.'
```

Comment:
I thought this is [spdy web compiler](https://github.com/swc-project)?

Reply:
Good point 😄 I will fix this!

Reply:
Actually, the 1.0 blog posts "(speedy web compiler) "

@kdy1 which do you prefer?

Reply:
It was `spdy web compiler`, but as many people said it's misleading.
So I cahnged it to `speedy web compiler`.